python/tk_desktop/desktop_engine_project_implementation.py

```diff
@@ -119,7 +119,7 @@ def _register_commands(self):
             self.__callback_map[("__commands", name)] = command_info["callback"]
             # pull out needed values since this needs to be pickleable
             gui_properties = {}
-            for prop in ["type", "icon", "title", "description"]:
+            for prop in ["type", "icon", "title", "description", "group", "group_default"]:
```

Comment:
Would be great if we could add these options to the docs here: http://developer.shotgunsoftware.com/tk-core/platform.html#sgtk.platform.Engine.register_command